src/inspector_js_api.cc

```diff
@@ -245,9 +245,9 @@ static void AsyncTaskScheduledWrapper(const FunctionCallbackInfo<Value>& args) {
   Environment* env = Environment::GetCurrent(args);
 
   CHECK(args[0]->IsString());
-
-  TwoByteValue task_name_buffer(args.GetIsolate(), args[0]);
-  StringView task_name_view(*task_name_buffer, task_name_buffer.length());
+  Local<String> task_name = args[0].As<String>();
+  String::Value task_name_value(args.GetIsolate(), task_name);
+  StringView task_name_view(*task_name_value, task_name_value.length());
```

Comment:
Do we need a full revert, I feel like this part can stay the same?

Reply:
A full revert is easier to work with. You're free to make incremental changes after it.

Reply:
A good illustration of why atomic PRs should always be preferred.